.github/workflows/bump-deps.yaml

```diff
@@ -0,0 +1,64 @@
+name: Bump DEPS
+on:
+  workflow_dispatch:
+    inputs:
+      dep:
+        description: "Dependency"
+        required: true
+        type: string
+        default: "vendor/web-discovery-project"
+      hash:
+        description: "New hash for dependency"
+        required: true
+        type: string
```

Comment:
If we give this a default empty value, we could then figure out the HEAD ref for the dependency and use that.

I'm guessing every time this is used, people will want to point to HEAD.

Reply:
This is now handled. If `ref` is not passed, it uses the newest commit hash from a branch called `master` or `main`. Doc comment updated in fb8a6159dc